curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/build-a-palindrome-checker-project/build-a-palindrome-checker.md

```diff
@@ -228,6 +228,30 @@ checkBtn.click();
 assert.strictEqual(resultEl.innerText.trim().replace(/[.,?!]+$/g, '').toLowerCase(), 'five|\_/|four is not a palindrome');
 ```
 
+When the `#text-input` element contains a palindrome created from a random sequence of alphanumeric characters, the `#result` element 
+should correctly identify it as a palindrome. 
```

Comment:
```suggestion
When the `#text-input` element contains an alphanumeric palindrome, the `#result` element should correctly identify it as a palindrome. 
```
Is it needed to mention how this palindrome was created?

Reply:
I guess not.